SerhiiBuza/HW7/task3.py

```diff
@@ -0,0 +1,12 @@
+def create_dict(st):
```

Comment:
good job
but look at more optimize version of code

good job
but look at more optimize code